CONTRIBUTING.md

```diff
@@ -20,6 +22,7 @@ If you don't care how and why and just want something that works, follow these s
 1. `cd $GOPATH/src/github.com/exercism/cli` (or `cd %GOPATH%/src/github.com/exercism/cli` on Windows)
 1. `git remote set-url origin https://github.com/<your-github-username>/cli`
 1. `go get -u github.com/golang/dep/cmd/dep`
+   * depending on your setup, you may need to install `dep` by following the instructions in the [`dep` repo](https://github.com/golang/dep)
```

Comment:
I think this still may be a bit confusing for new contributors. The `go get` command might work, but when it doesn't we don't actually explicitly say that its okay to fail and proceed with the `dep` install steps. What do you think, seeing as you are updating the documentation for this exact reason? 

Do you think if we move this step up to `2.` and make it read like below it would've simplified things for you? 

```
1. [Install dep](https://golang.github.io/dep/docs/installation.html)
```

Reply:
I see your point. I'm not sure what the setup needs to look like in order for the existing instruction to work as expected. For me the direct install from the `dep` link was necessary, but I'm not sure how many people would have the same experience.

What you've suggested (just including the link and forgetting about the `go get` command) is foolproof, so that change would make sense to me.

Reply:
I like the sound of something a bit more foolproof—getting stuff installed and configured is hard enough as it is 👍